ru/lessons/advanced/otp-supervisors.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: OTP Супервизоры
+category: advanced
+order: 6
+lang: ru
+---
+
+Супервизоры это специальные процессы с единственной целью: мониторинг других процессов. Супервизоры позволяют нам создавать отказоустойчивые приложения при помощи автоматического перезапуска дочерних процессов в случае их отказа.
+
+## Содержание
+
+- [Настройка](#section-1)
+  - [Стратегии](#section-2)
+  - [Вложенность](#section-3)
+- [Наблюдатель задач](#section-4)
+  - [Настройка](#section-5)
+  - [Наблюдаемые задачи](#section-6)
+
+## Настройка
+
+Магия Супервизоров заключена в функции `Supervisor.start_link/2`.  Помимо запуска нашего Супервизора и его потомков, она позволяет нам определять стратегию, которую будет использовать наш Супервизор для управления дочерними процессами.
```

Comment:
I have read it once more and decided to leave every "Supervisor" lowercase except first words of sentence and the one at the 42th line because there we're telling about "Supervisor" as a unique class. Could you, please, check it one more time and tell me whether we can assume this version final?